### ISSUE TITLE
Read Reek config

### ISSUE THREAD

**Issue by andyw8** [andyw8/codeclimate-reek#2]
Currently any Reek config file is ignored.

**Comment by troessner** [comment]
Yep, correct. An Examiner gets only passed a valid source. Determining what that source is happens wayyyy before creating an Examiner.

But I don't think that this is even an issue:

AFAIS the exclusion of paths is deprecated at CC in favor of specifying paths explicitly which you could just take from the CC config and feed it to the reek binary.
